tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Christoffel.cpp

```diff
@@ -39,4 +56,93 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.Christoffel.",
   test_christoffel<1, IndexType::Spacetime>(0.);
   test_christoffel<2, IndexType::Spacetime>(0.);
   test_christoffel<3, IndexType::Spacetime>(0.);
+
+  // Check that compute items work correctly in the DataBox
+  // First, check that the names are correct
+  CHECK(gr::Tags::SpacetimeChristoffelFirstKindCompute<3, Frame::Inertial,
+                                                       DataVector>::name() ==
+        "SpacetimeChristoffelFirstKind");
+  CHECK(
+      gr::Tags::TraceSpacetimeChristoffelFirstKindCompute<3, Frame::Inertial,
+                                                          DataVector>::name() ==
+      "TraceSpacetimeChristoffelFirstKind");
+  CHECK(gr::Tags::SpacetimeChristoffelSecondKindCompute<3, Frame::Inertial,
+                                                        DataVector>::name() ==
+        "SpacetimeChristoffelSecondKind");
+
+  // Check that the compute items return correct values
+  const DataVector test_vector{5., 4.};
+  MAKE_GENERATOR(generator);
+  std::uniform_real_distribution<> distribution(-0.1, 0.1);
+
+  auto spacetime_metric =
+      make_with_random_values<tnsr::aa<DataVector, 3, Frame::Inertial>>(
+          make_not_null(&generator), make_not_null(&distribution), test_vector);
```

Comment:
Same note as in #1478: if the `test_vector` is just used for size, then you can probably just input the numeric size you want instead.

Reply:
squashed